cf_xarray/accessor.py

```diff
@@ -168,6 +177,71 @@ def _get_groupby_time_accessor(var: Union[DataArray, Dataset], key: str) -> List
         return []
 
 
+def _get_custom_criteria(
```

Comment:
The docstring needs to be updated

Reply:
I made some updates in https://github.com/xarray-contrib/cf-xarray/pull/234/commits/0125789e332202755ea6c7ffcbd9999e6dfbb8ab